openc3/lib/openc3/microservices/decom_microservice.rb

```diff
@@ -100,6 +100,7 @@ def decom_packet(topic, msg_id, msg_hash, _redis)
           packet.extra = extra
         end
         packet.buffer = msg_hash["buffer"]
+        packet.process # Run processors
```

Comment:
So previously they were run on any buffer assignment but not it's only run explicitly. Did processors previously run multiple times in the packet processing chain? What are the side effects of this change?

Reply:
In COSMOS 5, they were being run in both the Interface and the Decom microservices before this change. 
Now they will only run once in Decom.
Side effects are if you needed them in the Interface/Protocol for some reason you will need to run them manuallly or add a "runProcessors" Protocol or something.  
Now processors can be used for things like sending commands, without having them run twice.